src/sxastarter/package.json

```
@@ -29,15 +29,16 @@
   },
   "license": "Apache-2.0",
   "dependencies": {
-    "@sitecore-jss/sitecore-jss-nextjs": "^20.1.0-canary",
+    "@sitecore-jss/sitecore-jss-nextjs": "ver21",
```

Let's use full version
```suggestion
    "@sitecore-jss/sitecore-jss-nextjs": "^21.0.0-canary",
```